### ISSUE TITLE
Segmentation fault (11) with 7.0.5-1~dotdeb+8.1

### ISSUE THREAD

**Issue by xstormy** [gplessis/dotdeb-php#128]
Hi,

we are getting Segmentation fault (11) using PHP 7.0.5-1~dotdeb+8.1 (apache2 2.4.10-10+deb8u4, libapache2-mod-php7.0 7.0.5-1~dotdeb+8.1, with mpm_prefork module).

What is your distribution? Debian 8 (8.4 Jessie)
What is your architecture? amd64
Which exact PHP version are you using? 7.0.5-1~dotdeb+8.1
Which SAPI are you using? mod_php
What’s the full content of your /etc/apt/sources.list and /etc/apt/sources.list.d files?

```
deb http://ftp.debian.org/debian jessie main non-free contrib
deb http://security.debian.org/ jessie/updates main non-free contrib

deb http://packages.dotdeb.org jessie all
deb-src http://packages.dotdeb.org jessie all
```

What is the result of dpkg -l 'php*' | grep '^.i' ?

```
ii  php-common      21-1~dotdeb+8.1      all          Common files for PHP packages
ii  php-readline    21-1~dotdeb+8.1      all          readline module for PHP [default]
ii  php-tcpdf       6.0.093+dfsg-1       all          PHP class for generating PDF files on-the-fly
ii  php7.0          7.0.5-1~dotdeb+8.1   all          server-side, HTML-embedded scripting language (metapackage)
ii  php7.0-apcu     5.1.3-1~dotdeb+8.1   amd64        APC User Cache for PHP
ii  php7.0-cli      7.0.5-1~dotdeb+8.1   amd64        command-line interpreter for the PHP scripting language
ii  php7.0-common   7.0.5-1~dotdeb+8.1   all          Common files for packages built from the PHP source
ii  php7.0-curl     7.0.5-1~dotdeb+8.1   amd64        CURL module for PHP
ii  php7.0-dbg      7.0.5-1~dotdeb+8.1   amd64        Debug symbols for PHP7.0
ii  php7.0-gd       7.0.5-1~dotdeb+8.1   amd64        GD module for PHP
ii  php7.0-json     7.0.5-1~dotdeb+8.1   amd64        JSON module for PHP
ii  php7.0-mcrypt   7.0.5-1~dotdeb+8.1   amd64        libmcrypt module for PHP
ii  php7.0-mysql    7.0.5-1~dotdeb+8.1   amd64        MySQL module for PHP
ii  php7.0-opcache  7.0.5-1~dotdeb+8.1   amd64        Zend OpCache module for PHP
ii  php7.0-readline 7.0.5-1~dotdeb+8.1   amd64        readline module for PHP
```

Please provide a textual description of the bug (don’t hesitate to provide some code, the expected result…).

Random Segmentation faults with log entries:

```
[core:notice] [pid 17587] AH00052: child pid 18181 exit signal Segmentation fault (11)
[core:notice] [pid 17587] AH00052: child pid 21123 exit signal Segmentation fault (11)
[core:notice] [pid 17587] AH00052: child pid 21213 exit signal Segmentation fault (11)
```

Please provide a backtrace, after installing php5-dbg.

```
[New LWP 27562]

warning: .dynamic section for "/usr/lib/php/20151012/curl.so" is not at the expected address (wrong library or version mismatch?)
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
Core was generated by `/usr/sbin/apache2 -k start'.
Program terminated with signal SIGSEGV, Segmentation fault.
#0  0x00007f35b8bfb0af in zend_declare_property_ex (ce=0x7f35b5811a40, name=0x7f35b8d3b8d1, property=0x7fffbdd793e8, access_type=0, doc_comment=0x7fffbdd79273)
    at /usr/src/builddir/Zend/zend_API.c:3643

warning: Source file is more recent than executable.
3643                            ce->default_properties_table = perealloc(ce->default_properties_table, sizeof(zval) * ce->default_properties_count, ce->type == ZEND_INTERNAL_CLASS);
(gdb) bt
#0  0x00007f35b8bfb0af in zend_declare_property_ex (ce=0x7f35b5811a40, name=0x7f35b8d3b8d1, property=0x7fffbdd793e8, access_type=0, doc_comment=0x7fffbdd79273)
    at /usr/src/builddir/Zend/zend_API.c:3643
#1  0x0000000000000000 in ?? ()
```

What cause the SIGSEGV? Thanks!

**Comment by gplessis** [comment]
Please let me know if the just-released 7.0.7 fixes the issue.

**Comment by gplessis** [comment]
Any feedback?
